framework/Assembler.py

```diff
@@ -43,7 +43,12 @@ def __init__(self):
     self.name = self.__class__.__name__  # name
     if not hasattr(self, 'assemblerObjects'): # protect against polyinheritance woes
       self.assemblerObjects = {}           # {MainClassName(e.g.Distributions):[class(e.g.Models),type(e.g.ROM),objectName]}
-      self.requiredAssObject = [False, ([], [])]
+      # _requiredAsmbObject = [check_number, [name_list], [number_list]]
+      #  where name_list is the tokens required (if check_number is True)
+      #  and number_list is a string list of either the number required
+      #   - number required (if optional) or 'n' if one or more required
+      #   or '-n' if not required.
```

Comment:
is this description accurate with the InputData.Quantity usage?

Reply:
Fixed.  Thanks.